dali/pipeline/operators/decoder/nvjpeg_decoder.cu

```diff
@@ -29,10 +29,14 @@ Output of the decoder is on the GPU and uses `HWC` ordering.)code")
   .AddOptionalArg("use_batched_decode",
       R"code(Use nvJPEG's batched decoding API.)code", false)
   .AddOptionalArg("device_memory_padding",
-      R"code(Padding for nvJPEG's device memory allocations.)code",
+      R"code(Padding for nvJPEG's device memory allocations.
+This parameter helps avoid reallocation in the nvJpeg whenever a bigger image
+is encountered and internal buffer needs to be reallocated to decode it. Default is 16MB.)code",
       16*1024*1024)
   .AddOptionalArg("host_memory_padding",
-      R"code(Padding for nvJPEG's host memory allocations.)code",
+      R"code(Padding for nvJPEG's host memory allocations.
+This parameter helps avoid reallocation in the nvJpeg whenever a bigger image
```

Comment:
"reallocation in nvJPEG" for both lines 33 and 38.

Reply:
Done